jscomp/bsb_exe/rescript_main.ml

```diff
@@ -84,13 +84,13 @@ let ninja_command_exit (type t) (ninja_args : string array) : t =
    ninja -C _build
 *)
 let clean_usage =
-  "Usage: rescript.exe clean <options>\n\n\
-   `rescript clean` only cleans the current project\n"
+  "Usage: rescript clean <options>\n\n\
+   `rescript clean` cleans build artifacts. It is useful for edge-case reasons in case you get into a stale build\n"
```

Comment:
Not sure about the wording here or if we should mention edge cases/stale builds at all. 

Maybe just
```suggestion
   `rescript clean` cleans build artifacts\n"
```
?

What do you think @cristianoc?

Reply:
That's cleaner

Reply:
Ok, then could you accept my change suggestion @DZakh?